addons/mass_mailing/models/mailing.py

```diff
@@ -330,6 +331,15 @@ def _compute_reply_to(self):
             elif mailing.reply_to_mode == 'update':
                 mailing.reply_to = False
 
+    @api.depends('mailing_model_id', 'mailing_domain')
+    def _compute_mailing_filter_count(self):
+        filter_data = self.env['mailing.filter']._read_group([
+            ('mailing_model_id', 'in', self.mapped('mailing_model_id').ids)
```

Comment:
`mapped` is useless on many2one

```suggestion
            ('mailing_model_id', 'in', self.mailing_model_id.ids)
```